DQMOffline/L1Trigger/python/L1TEGammaOffline_cfi.py

```diff
@@ -1,14 +1,15 @@
+from builtins import range
```

Comment:
BTW, just out of curiosity: Why this import instead of using `six.moves.range` (as you've used `six` before)?

Reply:
hi- I guess it was the first solution suggested by google in this case. Both solutions provide the eventual python3 behavior.